lib/Doctrine/Deprecations/Deprecation.php

```diff
@@ -56,6 +56,9 @@ class Deprecation
     private static $ignoredPackages = [];
 
     /** @var array<string,int> */
+    private static $ignoredLinksCount = [];
```

Comment:
the name looks weird to me here. Those counts will not be for *ignored* links (as ignored links will return before incrementing this)

Reply:
Oef, you're right. Changed it now, is this better?